conda_build/main_build.py

```diff
@@ -255,6 +255,7 @@ def execute(args, parser):
     import shutil
     import tarfile
     import tempfile
+    from os import makedirs
```

Comment:
I don't see a point in importing only specific functions from `os`. I would rather prefer to write `os.path.exists()` instead of `exists()` because the latter seems to me like a local function. Thus, I would vote for `import os` instead of a bunch of `from os.path import ...`